csmgr/csmgr.py

```diff
@@ -211,6 +213,38 @@ async def addcreator(
 
         await safe_add_role(ctx, member, self.cog_creator_role)
         await ctx.send(f"Done. {member.mention} is now a cog creator!")
+    
+    @is_core_dev_or_qa()
+    @commands.command()
+    async def removecreator(self, ctx: commands.Context, user: Union[discord.Member, int]) -> None:
+        """
+        Unregister a cog creator
+
+        User can be a user id if the user is not in the server
+        """
+        if isinstance(user, discord.Member):
+            user_id = user.id
+        else:
+            user_id = user
+        repos = await self.get_user_repos(user_id)
+        if not repos:
+            return await ctx.send("That user is not marked as a cog creator.")
+        
+        # Remove the user's roles, if they're still in the server
+        if isinstance(user, discord.Member):
+            await safe_remove_role(ctx, user, COG_CREATOR_ROLE_ID)
+            await safe_remove_role(ctx, user, SENIOR_COG_CREATOR_ROLE_ID)
+        
+        # Archive their support channel(s)
+        for repo in repos:
+            support_channel = repo.support_channel
+            if not support_channel:
+                continue
+            archive_cat = ctx.guild.get_channel(CHANNEL_ARCHIVE_ID)
```

Comment:
This could be made as a property (see `support_category_channel` property at line 138 for example).